automation/taskcluster/lib/variant.py

```diff
@@ -0,0 +1,20 @@
+class Variant:
+    def __init__(self, raw, abi, is_signed, build_type):
```

Comment:
If this function ever becomes too boilerplate-y, we could consider using http://www.attrs.org/en/stable/. It's not worth doing it, at the moment, I think.